index.js

```diff
@@ -586,7 +586,7 @@ export default async function makeHyperFetch ({
         const length = (end - start + 1)
 
         return {
-          status: 200,
+          status: 204,
```

Comment:
```suggestion
          status: 206,
```

I think we want "Partial COntent" and not "No Content" here.

Reply:
Oops, NVM, This is in HEAD